.github/compatibility-test-matrices/release-v4.4.x/transfer-chain-a.json

```diff
@@ -4,6 +4,7 @@
   ],
   "chain-b": [
     "release-v4.4.x",
+    "v7.1.0",
     "v7.0.1",
     "v6.2.0",
     "v6.1.1",
```

Comment:
versions on separate lines make these diff reviews way easier, is it possible to format the other files so we can see the diffs this way? (I'd be happy with a follow up PR so as to not make this one more difficult to review!)

Reply:
This does the trick locally (ran from the compatibility folder):

```bash
find . -name "*.json" -type f | while IFS= read -r json_file; do
    jq '.' "$json_file" > temp.json && mv temp.json "$json_file"
done
```

Reply:
Nice @DimitrisJim, flexing your scripting muscles! 💪 Would you like to open a PR with those changes?

Reply:
easy peasy lemon squeezy

Reply:
🍋